src/js_native_api_v8.cc

```diff
@@ -0,0 +1,2895 @@
+#include <limits.h>  // INT_MAX
+#include <cmath>
+#define NAPI_EXPERIMENTAL
+#include "js_native_api_v8.h"
```

Comment:
`js_native_api_v8.h` depends on the content of `js_native_api.h` so it didn't compile for me until I reversed these two lines.  It would be better if `js_native_api_v8.h` included `js_native_api.h`.

Reply:
ditto for this. Tons of compile errors because `js_native_api_v8.h` depends on `js_native_api.h`